Bottom: c5e2f20572c7a6b9d6005f2a8dfa458b09b156fd
Top:    9e9da779f337dc3d381d204c0c83ddbbd6bce592
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-14 15:58:18 -0500

Refresh of gitlab-auth

---
diff --git a/roles/buildbot-master/defaults/main.yml b/roles/buildbot-master/defaults/main.yml
index 9e22c2e6..52d4e40a 100644
--- a/roles/buildbot-master/defaults/main.yml
+++ b/roles/buildbot-master/defaults/main.yml
@@ -44,8 +44,8 @@ container_env:
   BUILDBOT_WORKER_PTXDIST: 'worker-ptxdist'
   BUILDBOT_WORKER_PTXDIST_PASS: '{{ buildbot_worker_ptxdist_password }}'
   GITLAB_URI: 'https://{{ gitlab_hostname }}/'
-  GITLAB_CLIENT_ID: '{{ gitlab_buildbot_auth_id }}'
-  GITLAB_CLIENT_SECRET: '{{ gitlab_buildbot_auth_secret }}'
+  GITLAB_OAUTH_ID: '{{ gitlab_buildbot_auth_id }}'
+  GITLAB_OAUTH_SECRET: '{{ gitlab_buildbot_auth_secret }}'
   LDAP_URI: 'ldap://{{ openldap_hostname }}:389/'
   LDAP_BIND_USER: 'cn=proxyagent,dc=novatech'
   LDAP_BIND_PW: '{{ openldap_proxyagent_password }}'
